Fix name into xml files to merge two author page

riyadomf opened the pull request:
Fixes #3606 
I have two papers with different first name, last name combination. Because of the variation two author page have been created. I want to merge the second one into the first author page
https://aclanthology.org/people/m/md-omar-faruqe/
https://aclanthology.org/people/o/omar-riyad/

nschneid commented:
Thanks for bringing this up. Both of the PDFs actually say Omar Faruqe Riyad. Please update the XML under https://github.com/acl-org/acl-anthology/tree/master/data/xml to match this, and then there will be no need for the name variants change.

riyadomf commented:
> Thanks for bringing this up. Both of the PDFs actually say Omar Faruqe Riyad. Please update the XML under https://github.com/acl-org/acl-anthology/tree/master/data/xml to match this, and then there will be no need for the name variants change.

Thanks for your reply. I have updated both xml to match the name and removed the name variant change.
